### PR TITLE
feat: lo-fi contact page

### PR DESCRIPTION
**Pull request by katyhonguyen**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

Contact page Lo-fi. Haven't implemented the icons.

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

IMG_5491

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @insert pl github username here

### CODE REVIEW

File: styles.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: 'white',
+    borderRadius: 20,
+  },
+
+  contactText: {
+    color: '#4F4F4F',
+    fontSize: 14,
+    marginBottom: 50,
```
Comment (carolynzhuang) [inline styles.ts:60]: I think it's better to try to avoid using margins on text

File: styles.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    fontSize: 32,
+    fontWeight: '700',
+    paddingLeft: 29,
+    paddingBottom: 10,
+    textAlign: 'left',
+  },
+
+  imageContainer: {
+    width: '100%',
+    height: width * (8 / 9),
```
Comment (carolynzhuang) [inline styles.ts:23]: Maybe use aspect ratio here?

File: styles.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  },
+
+  responsiveImage: {
+    width: '100%',
+    height: '100%',
+    resizeMode: 'cover',
+  },
+
+  overlay: {
+    position: 'absolute',
```
Comment (carolynzhuang) [inline styles.ts:34]: Using absolute positioning might not show up well on different screen sizes

File: styles.ts [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+const { width } = Dimensions.get('window');
+
+export const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: 'column',
+    backgroundColor: 'white',
+  },
+
+  Heading4: {
```
Comment (carolynzhuang) [inline styles.ts:12]: I feel like this name could be more descriptive

File: App.tsx
Comment (carolynzhuang) [inline App.tsx:1]: i think this will eventually be moved to contactPage in pages?

File: styles.ts
Comment (carolynzhuang) [inline styles.ts:1]: move to src/pages

File: styles.ts (hunk 4 above)
Comment (carolynzhuang) [inline styles.ts:12]: This name could be more descriptive

File: styles.ts (hunk 2 above)
Comment (carolynzhuang) [inline styles.ts:23]: Maybe use aspect ratio here?

File: styles.ts (hunk 3 above)
Comment (carolynzhuang) [inline styles.ts:34]: Absolute positioning might not show up well on different screen sizes

File: styles.ts [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  contactText: {
+    color: '#4F4F4F',
+    fontSize: 14,
+    marginBottom: 50,
+  },
+
+  managerText: {
+    color: '#4F4F4F',
+    fontSize: 14,
+    marginBottom: 10,
```
Comment (carolynzhuang) [inline styles.ts:66]: change to padding

**Review by christophertorres1 — CHANGES_REQUESTED** [review]
Minor change for file structure consistency. Update to the current design and ignore further changes made to the contact screen for now. Wonderful job Katy!

**Review by christophertorres1 — APPROVED** [review]
